client/rest-high-level/src/main/java/org/elasticsearch/client/RestHighLevelClient.java

```diff
@@ -121,6 +123,22 @@ public void indexAsync(IndexRequest indexRequest, ActionListener<IndexResponse>
         performRequestAsyncAndParseEntity(indexRequest, Request::index, IndexResponse::fromXContent, listener, emptySet(), headers);
     }
 
+    /**
+     * Deletes a document by id using the delete api
```

Comment:
Right. I did not notice the modifications when I merged the master branch into my branch.